### ISSUE TITLE
windowSize inconsistency

### ISSUE THREAD

**Issue by bevacqua** [zeit/hyperterm#430]
The `windowSize` property expects `px/px`, but when you resize a terminal window a different format _(don't know which)_ is reported.

![screen shot 2016-07-26 at 16 21 20](https://cloud.githubusercontent.com/assets/934293/17152151/0190e712-534d-11e6-90a7-1c82d1c2284e.png)

**Comment by matheuss** [comment]
I think that `windowSize` should move to columns, which is the reported when you resize the window 😅

**Comment by albinekb** [comment]
I think it would be best to make it possible to support both px and columns, since columns is based on font-size

**Comment by bevacqua** [comment]
I think default measure should be whatever is reported when resizing. Then you can have whatever complicated stuff you want behind that sensible default foundation 😄

**Comment by Evanlec** [comment]
Agreed, let us use columns by default for the windowSize property, as that is familiar to terminal users. And it is seen easily when resizing. Also, I would think px makes it difficult to determine what the window size will be when using different displays (retina for example).

**Comment by rauchg** [comment]
I think we can keep `windowSize` as is, and introduce `termSize` in rows / cols. This way we don't throw away backwards compatibility, and truthfully cols and rows are not the size of the window, they're the size of the inner terminal.

🎉

**Comment by JacobDB** [comment]
I actually can't even get windowSize to work at all :/

**Comment by bevacqua** [comment]
@JacobDB Works for me. Make sure to place it under `config` (this isn't very well documented imo)

https://github.com/bevacqua/dotfiles/blob/9175d6ae51f4fdd941b56a6f9deddd8c10dfeed0/hyperterm/.hyperterm.js#L7

**Comment by JacobDB** [comment]
@bevacqua thanks! Yea, I was trying to add it as a new entry, didn't realize it had to be under config. It's working now :)

**Comment by chabou** [comment]
thanks to xterm.js in our v2, it could be easier to implement this
